lte/gateway/c/session_manager/OperationalStatesHandler.cpp

```diff
@@ -37,13 +38,16 @@
 
 namespace magma {
 
-OpState get_operational_states(magma::SessionStore* session_store) {
+OpState get_operational_states(magma::lte::SessionStore* session_store) {
   std::list<std::map<std::string, std::string>> states;
   auto session_map = session_store->read_all_sessions();
+  folly::dynamic subscribers = folly::dynamic::object;
```

Comment:
Out of scope for this PR, but we did migrate away from using Folly for JSON generation in MME a few months back (I think @LKreutzer worked on this?) 
Maybe this is something we can consider for SessionD as well and leave Folly for the event management only? Just something to consider, I am also fine with just leaving it alone :D

Reply:
@themarwhal We have already discussed it and have decided to remove Folly dependencies for SessionD in a separate PR .